num/src/divrem.rs

```diff
@@ -0,0 +1,5 @@
+pub trait DivRem {
```

Comment:
You may stick header in this file mantioning you as the author (you can take a sample from headers in other files)

Reply:
Ok, thanks